### PR TITLE
Fix breadcrumbs on claim status

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4326

This fixes the breadcrumbs for the claim status app. It looks like there were some generic breadcrumbs added for Appeals, but the claims pages have some nested page and need to generate specific links for breadcrumbs. So I've moved the global breadcrumbs into an appeals specific layout component for those pages.

screen shot 2017-09-07 at 1 51 37 pm
screen shot 2017-09-07 at 1 51 09 pm
screen shot 2017-09-07 at 1 50 50 pm
screen shot 2017-09-08 at 9 16 22 am

### CODE REVIEW

File: src/js/disability-benefits/components/AppealLayout.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Breadcrumbs from './Breadcrumbs';
+
+export default function AppealLayout({ children }) {
+  return (
+    <div>
+      <div>
+        <div className="row">
```
Comment (cvalarida) [inline src/js/disability-benefits/components/AppealLayout.jsx]: Why is this nested in an otherwise empty `div`?

File: src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx [hunk 2]
```diff
@@ -30,11 +29,6 @@ class DisabilityBenefitsApp extends React.Component {
         loginUrl={this.props.loginUrl}
         verifyUrl={this.props.verifyUrl}>
         <AppContent>
-          <div>
-            <div className="row">
-              <Breadcrumbs location={this.props.location}/>
```
Comment (cvalarida) [inline src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx]: Why was this removed?

**Review by cvalarida — APPROVED** [review]
The screenshots look good, but I had a few question.

File: src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx (hunk 2 above)
Comment (cehsu) [reply src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx]: It looks like it's included in the children now instead.

File: src/js/disability-benefits/components/AppealLayout.jsx (hunk 1 above)
Comment (cehsu) [reply src/js/disability-benefits/components/AppealLayout.jsx]: Should the classes of the nested `div`s be consistent with other usages of Breadcrumbs?
```
<div>
  <div className="row">
    <div className="medium-12 columns">
      <Breadcrumbs/>
    </div>
  </div>
  <div className="row">
```

File: src/js/disability-benefits/routes.jsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ import AskVAPage from './containers/AskVAPage.jsx';
 import DocumentRequestPage from './containers/DocumentRequestPage.jsx';
 import AdditionalEvidencePage from './containers/AdditionalEvidencePage.jsx';
 import ClaimEstimationPage from './containers/ClaimEstimationPage.jsx';
+import AppealLayout from './components/AppealLayout';
```
Comment (cehsu) [inline src/js/disability-benefits/routes.jsx]: It looks like AppealLayout is containing a container, should it also be a container rather than a component (located in the component directory)?

File: src/js/disability-benefits/containers/YourClaimsPage.jsx [hunk 4]
```diff
@@ -173,14 +174,19 @@ class YourClaimsPage extends React.Component {
 
     return (
       <div className="your-claims">
+        <Breadcrumbs/>
```
Comment (cehsu) [inline src/js/disability-benefits/containers/YourClaimsPage.jsx]: It looks like the class nesting is slightly different here. Does the `your-claims` class serve a styling purpose? Does the specificity override other styling?

**Review by cehsu — APPROVED** [review]
LGTM.

File: src/js/disability-benefits/components/AppealLayout.jsx (hunk 1 above)
Comment (jbalboni) [reply src/js/disability-benefits/components/AppealLayout.jsx]: I am not sure why I added that extra div. Probably can remove it.

@cehsu I was matching the way the Appeals app was using it. The layout is a little finicky, so I think I'd like to leave it the way it is since it looks right.

File: src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx (hunk 2 above)
Comment (jbalboni) [reply src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx]: The breadcrumbs are now either in specific page components (claims) or in the AppealsLayout (appeals).

File: src/js/disability-benefits/routes.jsx (hunk 3 above)
Comment (jbalboni) [reply src/js/disability-benefits/routes.jsx]: I think since it's a regular component it should go in components.

File: src/js/disability-benefits/containers/YourClaimsPage.jsx (hunk 4 above)
Comment (jbalboni) [reply src/js/disability-benefits/containers/YourClaimsPage.jsx]: Yeah, `your-claims` overwrites some styles in its children (a lot of the css for this app is a tangled mess). I'm going to create a tech debt ticket for that.

File: src/js/disability-benefits/components/AppealLayout.jsx (hunk 1 above)
Comment (jbalboni) [reply src/js/disability-benefits/components/AppealLayout.jsx]: I tried add the columns div, but it adds more padding and affects the alignment, so I'd have to correct for it again here, which doesn't seem like an improvement.

File: src/js/disability-benefits/components/Breadcrumbs.jsx [hunk 5]
```diff
@@ -2,14 +2,13 @@ import React from 'react';
...
-    ];
-
-    return (<div className="db-breadcrumbs">
-      {crumbs.reduce((content, e) => { return [...content, ' › ', e]; }, []).slice(1)}
-    </div>);
+    return (<nav className="va-nav-breadcrumbs">
+      <ul className="row va-nav-breadcrumbs-list columns claims-breadcrumbs" role="menubar" arialLabel="Primary">
+        <li><a href="/" key="home">Home</a></li>
+        <li><a href="/disability-benefits" key="disability-benefits">Disability Benefits</a></li>
+        {this.props.children}
```
Comment (bshyong) [inline src/js/disability-benefits/components/Breadcrumbs.jsx]: 👍

**Review by bshyong — APPROVED** [review]
LGTM
